### ISSUE TITLE
train the data with sparse labels.

### ISSUE THREAD

**Issue by wpl18589946597** [suraj-deshmukh/Multi-Label-Image-Classification#10]
I used your code to train the data with sparse labels. I found that the training effect of the model is not good. The proportion of the test data that completely correct is too small . I guess the loss function or the evaluation function is unreasonable. What good solution do you have?

**Comment by suraj-deshmukh** [comment]
the network configuration and loss function worked for my problem statement that doesn't necessarily mean it should work for any type of problem statement. Try with bigger network but loss function won't change as it is a multi label classification matrix and try with different optimization technique and you can add your own evaluation metric while tranining.

for your reference:
https://keras.io/metrics/
